### DISCUSSION TITLE
Issue with Initialization Process When Not Using TypeScript

### DISCUSSION THREAD

**Discussion by Miguel-Ubilla** [shadcn-ui/ui#1150] [Q&A]
Dear developers,

I have been trying to use shadcn-ui with Vite ReactJS. During the setup, I specifically selected that I would not be using TypeScript. However, I encountered an error related to the missing tsconfig.json. To address this, I created a placeholder tsconfig.json file at the root of the project, which allowed me to proceed. However, I believe this error shouldn't occur if I have opted not to use TypeScript. Below is the sequence of prompts and responses I encountered:
`npx shadcn-ui@latest init
✔ Would you like to use TypeScript (recommended)? … no / yes  (no)
✔ Which style would you like to use? › Default
✔ Which color would you like to use as base color? › Slate
✔ Where is your global CSS file? … src/app/globals.css
✔ Would you like to use CSS variables for colors? … no / yes
✔ Where is your tailwind.config.js located? … tailwind.config.js
✔ Configure the import alias for components: … @/components
✔ Configure the import alias for utils: … @/lib/utils
✔ Are you using React Server Components? … no / yes
✔ Write configuration to components.json. Proceed? … yes

✔ Writing components.json...
Failed to load tsconfig.json. Couldn't find tsconfig.json
`
I believe it would be beneficial for other users if this issue was addressed in future releases. Thank you for your hard work on this library and for considering this feedback.

**Comment by ayodhyaw2002** [comment]
Make sure your jsconfig.json file is located under the main folder. 
![image](https://github.com/shadcn-ui/ui/assets/118840840/d189595d-b2f2-4750-ac19-a872a06bc1ba)
